compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/semantics/analyzer/SymbolEnter.java

```diff
@@ -4188,6 +4321,66 @@ private void defineInvokableSymbolParams(BLangInvokableNode invokableNode, BInvo
         invokableSymbol.type.tsymbol.type = invokableSymbol.type;
     }
 
+    private List<BVarSymbol> defineParametersOfFunction(BLangInvokableNode invokableNode, SymbolEnv invokableEnv) {
```

Comment:
There seems to be quite a bit of code duplicated between this and defineParametersOfFunctionType. Shall we check and refactor?